argumentlist.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import "strings"
+
+type argumentList []string
+
+func (l argumentList) String() string {
+	return strings.Join(l, ",")
+}
+
+func (a *argumentList) Set(value string) error {
+	*a = append(*a, strings.Split(value, ",")...)
+	return nil
```

Comment:
Would returning an error when `value` is empty make sense ? (an empty string would be added to the `argumentList`)